201-managed-application-usingmsi-for-management/artifacts/ManagedAppZip/createUiDefinition.json

```diff
@@ -0,0 +1,39 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/0.1.2-preview/CreateUIDefinition.MultiVm.json#",
+  "handler": "Microsoft.Azure.CreateUIDef",
+  "version": "0.1.2-preview",
+  "parameters": {
+    "basics": [],
+    "steps": [
+      {
+        "name": "applicationSettings",
+        "label": "Application settings",
+        "subLabel": {
+          "preValidation": "Configure the application settings",
+          "postValidation": "Done"
+        },
+        "bladeTitle": "Application settings",
+        "elements": [
+          {
+            "name": "storageAccount",
+            "type": "Microsoft.Common.TextBox",
+            "label": "Storage Account",
+            "toolTip": "Specify storage account name",
+            "constraints": {
+              "required": true,
+              "regex": "^[a-z0-9A-Z]{1,30}$",
+              "validationMessage": "Only alphanumeric characters are allowed, and the value must be 1-30 characters long."
+            },
+            "visible": true
+          }
+        ]
+      }
+    ],
+    "outputs": {
+      "storageAccountName": "[steps('applicationSettings').storageAccount.name]",
+      "logicAppName": "[steps('applicationSettings').storageAccount.name]",
+      "rolenameguid": "[guid()]",
```

Comment:
remove this and move guid generation to the template

Reply:
Fixed